app/javascript/src/PaymentGateways/braintree/BraintreeForm.jsx

```diff
@@ -54,12 +56,23 @@ const BraintreeForm = ({
     }
   }, [braintreeNonceValue])
 
+  const get3DSecureError = (response) => {
+    return (
+      response.name === 'BraintreeError' ||
```

Comment:
I still don't think we need to check this. It either has `authenticate_(attempt_)?successful` or it's an error, no?

Reply:
if it reduces to that, then it's not needed at all